styles.css

```diff
@@ -59,7 +59,7 @@
 
 .welcome {
   position: static;
-  font-family: Helvetica, Arial, Verdana, Tahoma, sans-serif;
+  font-family: 'Crete Round', serif;;
```

Comment:
- [ ] Thanks for adding the font in your CSS, but it seems to not be rendering on the page(see the second screenshot below), this is because you haven't added the actual font to your `<head>` tag, I also left a screenshot on how to use google fonts below, and here is the [Crete Round font on google fonts](https://fonts.google.com/specimen/Crete+Round), so kindly go there, click select the style, and copy the link to the `<head>` like the aforementioned screenshot describes.
![image](https://user-images.githubusercontent.com/73140461/143493954-2fe9b23d-891d-49f2-ac4e-5eebc731f4f8.png)
![image](https://user-images.githubusercontent.com/73140461/143493802-50605cb0-443c-4fc4-9ccd-d66038b252df.png)